VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/participants-panel/participants-panel.component.spec.ts

```diff
@@ -24,7 +24,8 @@ import {
     hearingTransferSubjectMock,
```

Comment:
I'm not sure if I just can't see it on GitHub because the file is too big but is there a test missing for participants updated?